src/common/formatFamiliesData.ts

```diff
@@ -1,28 +1,77 @@
 import { Person } from "@/components/Form/formFunctions";
 import { Schema } from "@/databaseUtils";
 import { displayList } from "@/common/format";
+import { getChildAgeUsingBirthYearAndMonth, isChildUsingBirthYear } from "@/common/getAgesOfFamily";
+import { getCurrentYear } from "@/common/date";
 
 export interface HouseholdSummary {
     householdSize: string;
     genderBreakdown: string;
+    ageAndGenderOfAdults: string;
     numberOfBabies: string;
     ageAndGenderOfChildren: string;
 }
 
 const getChild = (child: Person): string => {
-    const gender = child.gender === "male" ? "M" : child.gender === "female" ? "F" : "-";
-    return `${child.age} ${gender}`;
+    let gender;
+    switch (child.gender) {
+        case "male":
+            gender = "M";
+            break;
+        case "female":
+            gender = "F";
+            break;
+        case "other":
+            gender = "-";
+            break;
+    }
+
+    const age = getChildAgeUsingBirthYearAndMonth(child.birthYear, child.birthMonth ?? null, true);
+
+    return `${age} ${gender}`;
+};
+
+const getAdult = (adult: Person): string => {
+    let gender;
+    switch (adult.gender) {
+        case "male":
+            gender = "M";
+            break;
+        case "female":
+            gender = "F";
+            break;
+        case "other":
+            gender = "-";
+            break;
+    }
+    const age = getCurrentYear() - adult.birthYear;
+    return `${age} ${gender}`;
```

Comment:
Is it possible to get the age outside of these functions and then generalise them? I'd probably move the age line to where we call this and pass in the value as a parameter

```
const getPerson = (person: Person, age: string): string => {
    let gender;
    switch (person.gender) {
        case "male":
            gender = "M";
            break;
        case "female":
            gender = "F";
            break;
        case "other":
            gender = "-";
            break;
    }
    return `${age} ${gender}`;

    ...

    ageAndGenderOfAdults: displayList(formattedAdults.map((adult) => getPerson(adult, getCurrentYear() - adult.birthYear))),
    ageAndGenderOfChildren: displayList(formattedChildren.map((child) => getPerson(child, getChildAgeUsingBirthYearAndMonth(child.birthYear, child.birthMonth ?? null, true))))
};
```

Reply:
Yep sounds good